server/viame_server/serializers/viame.py

```diff
@@ -210,7 +216,13 @@ def export_tracks_as_csv(
     csvFile = io.StringIO()
     writer = csv.writer(csvFile)
     if header:
-        writeHeader(writer)
+        metadata = None
+        if fps is not None:
+            metadata = {"fps": fps}
+            # TODO: REMOVE THIS RESET ONCE FIXED IN THE MAIN CONTAINER
+            # It is only here to enable training to work on videos because of a timestamp issue
+            fps = None
```

Comment:
```suggestion
        metadata = {}
        if fps is not None:
            metadata["fps"] = fps
            # TODO: REMOVE THIS RESET ONCE FIXED IN THE MAIN CONTAINER
            # It is only here to enable training to work on videos because of a timestamp issue
            fps = None
```

Will make it simpler to add more metadata.